mosh/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_name=mosh
+pkg_origin=core
+pkg_version=1.2.5
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('OCB-An-Authenticated-Encryption-Scheme')
```

Comment:
`pkg_license` should be `GPL-3.0`: https://github.com/mobile-shell/mosh/blob/master/COPYING